api/v1alpha1/vaultpkisecret_types.go

```diff
@@ -130,7 +132,6 @@ type VaultPKISecret struct {
 func (v *VaultPKISecret) GetIssuerAPIData() map[string]interface{} {
 	m := map[string]interface{}{
 		"common_name":          v.Spec.CommonName,
-		"alt_names":            v.Spec.AltNames,
 		"ip_sans":              v.Spec.IPSans,
 		"uri_sans":             v.Spec.URISans,
 		"other_sans":           v.Spec.OtherSans,
```

Comment:
It looks like we may need to do the same for ip_sans, uri_sans, and other_sans, since they also need to be comma-separated lists: https://developer.hashicorp.com/vault/api-docs/secret/pki#ip_sans

And on a related note, `VaultPKISecretSpec.OtherSans` may also need to be updated to be a `[]string` in vaultpkisecret_types.go, to follow the other fields.

Reply:
Good call!